bld/namelist_files/namelist_defaults_pop.xml

```diff
@@ -1392,6 +1394,7 @@
 <feventflux_input%filename ocn_grid="gx3v7">ocn/pop/gx3v7/forcing/feventflux_gx3v7_5gmol_cesm1_97_2017.nc</feventflux_input%filename>
 <feventflux_input%filename ocn_grid="gx1v6">ocn/pop/gx1v6/forcing/feventflux_gx1v6_5gmol_cesm1_97_2017.nc</feventflux_input%filename>
 <feventflux_input%filename ocn_grid="gx1v7">ocn/pop/gx1v6/forcing/feventflux_gx1v6_5gmol_cesm1_97_2017.nc</feventflux_input%filename>
+<feventflux_input%filename ocn_grid="tx0.1v3">ocn/pop/tx0.1v3/tmp/fesedflux_total_reduce_oxic_POP_tx0.1v3.c200420.nc</feventflux_input%filename>
```

Comment:
Need to either move this to proper directory (and import it to inputdata repo) or generate updated version